src/gen/src/AmBeSource.cc

```diff
@@ -129,7 +130,9 @@ AmBeSource::AmBeSource() {
   // about 25% of Am-Be reactions do not emit a 4.43 MeV gamma (depends on the
   // energy level the carbon atom is excited to after the (alpha,n) reaction)
   // This probability can be changed here is you desire
-  double prob_gamma_emission = 0.75;  // probability of emitted a gamma along with a neutron
+  DBLinkPtr neutrondb = DB::Get()->GetLink("AMBE_NSPECTRUM", "ambe_n");
+  double prob_gamma_emission = neutrondb->GetD("prob_gamma_emission");
+  // double prob_gamma_emission = 0.75;  // probability of emitted a gamma along with a neutron
```

Comment:
You can remove this comment.

Reply:
Apologies, I meant you can remove the commented out line 129:
  // double prob_gamma_emission = 0.75;  // probability of emitted a gamma along with a neutron